.circleci/config.yml

```
@@ -157,6 +157,104 @@ references:
         python3 --version
         demisto-sdk --version
 
+  download_configuration: &download_configuration
+    run:
+      name: Download Configuration
+      when: always
+      command: |
+        if ./Tests/scripts/is_ami.sh ;
+          then
+            ./Tests/scripts/download_demisto_conf.sh
+
+          else
+            ./Tests/lastest_server_build_scripts/download_demisto_conf.sh
+        fi
+  
+  test_download_upload: &test_download_upload
+    run:
+      name: Test Download and Upload commands
+      when: on_success
+      command: |
+        echo "Running upload"
+        demisto-sdk upload -i Packs/HelloWorld/Integrations/ --insecure > artifacts/upload.log
```

do this for all logs
```suggestion
        demisto-sdk upload -i Packs/HelloWorld/Integrations/ --insecure > ${CIRCLE_ARTIFACTS}/upload.log
```